minimal/content/pages/blog/category/react.md

```diff
@@ -0,0 +1,5 @@
+---
+layout: PostCategoryLayout
+title: React
```

Comment:
Not sure the title is being picked up correctly in the Studio which makes navigating the categories very confusing.

<img width="868" alt="Screen Shot 2021-11-22 at 10 46 19 am" src="https://user-images.githubusercontent.com/1386506/142786358-05e94519-38e7-4481-87c4-322fefd323ac.png">

Also after creating a category using the red + button I land on the category page which is empty, and the side panel is empty... I realise this may be the expected behaviour of a category, but it just feels incomplete when using it as a user in the Studio.

<img width="1619" alt="Screen Shot 2021-11-22 at 10 54 38 am" src="https://user-images.githubusercontent.com/1386506/142786512-19dd086f-e2f8-42f2-a05c-30a0b615b9ca.png">

Reply:
Hmmm, might be a bug, @TomasBankauskas as you work on the designs of the categories and post feeds can you add `labelField: title` to the PostCategoryLayout model.